test/Microsoft.DocAsCode.Dfm.Tests/DocfxFlavoredMarkdownTest.cs

```diff
@@ -369,11 +369,26 @@ public void TestDfmVideo_Video()
         {
             // 1. Prepare data
             var root = @"The following is video.
-> [!Video https://sec.ch9.ms/ch9/4393/7d7c7df7-3f15-4a65-a2f7-3e4d0bea4393/Episode208_mid.mp4]
+> [!Video https://channel9.msdn.com/Shows/This+Week+On+Channel+9/TWC9-Mad-about-Mads-and-C-80-HoloLens-Wins-Award-3D-Printed-Warthog-and-more/player]
 ";
 
             var expected = @"<p>The following is video.</p>
-<div class=""embeddedvideo""><iframe src=""https://sec.ch9.ms/ch9/4393/7d7c7df7-3f15-4a65-a2f7-3e4d0bea4393/Episode208_mid.mp4"" frameborder=""0"" allowfullscreen=""true""></iframe></div>
+<div class=""embeddedvideo""><iframe src=""https://channel9.msdn.com/Shows/This+Week+On+Channel+9/TWC9-Mad-about-Mads-and-C-80-HoloLens-Wins-Award-3D-Printed-Warthog-and-more/player?nocookie=true"" frameborder=""0"" allowfullscreen=""true""></iframe></div>
+";
+
+            var marked = DocfxFlavoredMarked.Markup(root);
+            Assert.Equal(expected.Replace("\r\n", "\n"), marked);
+        }
+        [Fact]
```

Comment:
new line?

Reply:
This is the same code being used in the other video related tests. It seems to just be a difference between how the expected string is constructed and the real markdown output gets created.